### PR TITLE
Upgrade font awesome v4.7

### PR DESCRIPTION
**Pull request by atomita**
Hi there,

Thank you for develop "iconfonts".

I was Upgrade font-awesome to v4.7
Please consider the merge.

Thanks

### CODE REVIEW

File: support/stylus.tpl [hunk 1]
```diff
@@ -97,5 +97,5 @@ $<%= prefix %>-icon
 
 <%= prefix %>-icons = {
 <% each(function (val, key) { %>
-  <%= key %>: "<%= val %>"<% }); %>
+  '<%= key %>': "<%= val %>"<% }); %>
```
Comment (rstacruz) [inline support/stylus.tpl]: have you confirmed that this works?

**Comment by rstacruz** [comment]
iconfonts@0.12.0 is out with these changes. thanks for your help!

**Comment by rstacruz** [comment]
I've added you to this repo. Have a read at git.io/col :)

File: support/stylus.tpl (hunk 1 above)
Comment (atomita) [reply support/stylus.tpl]: Yes, it is working.

**Comment by atomita** [comment]
@rstacruz Thank you for merge!

**Pull request merged**